test/ctia/stores/es/crud_test.clj

```diff
@@ -202,19 +212,22 @@
                             {}))))))
 
 (deftest make-search-query-test
-  (let [es-conn-state (-> (init/init-es-conn! props-not-aliased)
+  (let [app (helpers/get-current-app)
+        services (es-helpers/app->ESConnServices app)
+
+        es-conn-state (-> (init/init-es-conn! props-not-aliased services)
                           (update :props assoc :default_operator "AND"))
         simple-access-ctrl-query {:terms {"groups" (:groups ident)}}]
     (with-redefs [find-restriction-query-part (constantly simple-access-ctrl-query)]
       (let [query-string "*"
             es-query-string-AND {:query_string {:query query-string
                                                 :default_operator "AND"}}
             es-query-string-no-op {:query_string {:query query-string}}
-            date-range {:created {:gte "2020-04-01T00:00:00.000Z"
-                                  :lt "2020-05-01T00:00:00.000Z"}}
+            date-range {:created {:gte #inst "2020-04-01T00:00:00.000Z"
+                                  :lt #inst "2020-05-01T00:00:00.000Z"}}
             es-date-range {:range date-range}
-            filter-map {"disposition" 2
-                        "observable.type" "domain"}
+            filter-map {:disposition 2
+                        :observable.type "domain"}
```

Comment:
Update these tests to conform to existing schemas (this PR enables schema checking for this namespace).